.travis.yml

```diff
@@ -24,23 +24,10 @@ env:
 
 matrix:
   include:
-    - env: CONFIG=linux_ppc64le_python3.6 UPLOAD_PACKAGES=True DOCKER_IMAGE=ibmcom/powerai:1.6.2-pytorch-ubuntu18.04-py36-ppc64le
+    - env: CONFIG=linux_noarch UPLOAD_PACKAGES=True DOCKER_IMAGE=conda-forge/linux-anvil-ppc64le
```

Comment:
`conda-forge` should be `condaforge`. That is the cause of the travis failure.

Reply:
Thanks Billy! Sometimes, I think copy paste is better :)